dspace/config/dspace.cfg

```diff
@@ -810,7 +810,7 @@ event.dispatcher.RelatedItemEnhancerUpdatePoller.class = org.dspace.event.BasicD
 # Add rdf here, if you are using dspace-rdf to export your repository content as RDF.
 # Add iiif here, if you are using dspace-iiif.
 # Add orcidqueue here, if the integration with ORCID is configured and wish to enable the synchronization queue functionality
-event.dispatcher.default.consumers = versioning, discovery, eperson, dedup, crisconsumer, orcidqueue, audit, qaeventsdelete, referenceresolver, orcidwebhook, itemenhancer, customurl, iiif, reciprocal, filetypemetadataenhancer, authoritylink, licensegenerator, accesstype, authoritymetadataenhancer, formfieldcleaner
+event.dispatcher.default.consumers = versioning, discovery, eperson, dedup, crisconsumer, orcidqueue, audit, qaeventsdelete, referenceresolver, orcidwebhook, itemenhancer, customurl, iiif, reciprocal, filetypemetadataenhancer, authoritylink, accesstype, authoritymetadataenhancer, formfieldcleaner
```

Comment:
You could keep `licensegenerator` and modify behavior of this consumer:
* Check consumer event is correct --> a bitstream and acceptedActions.
* Check "default.licence.enabled" and have "default.license.value".
* Check bitstream doesn't have already a license --> if already a licens, then skip
* Add the default license.

As it, this consumer could be part of "uclouvain-base" branch